std/math.d

```diff
@@ -6832,6 +6832,149 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     assert(approxEqual(pow(twoI, three), 8.0));
 }
 
+/** Computes the value of a positive integer `x`, raised to the power `n`, modulo `m`.
+ *
+ *  Params:
+ *      x = base
+ *      n = exponent
+ *      m = modulus
+ *
+ *  Returns:
+ *      `x` to the power `n`, modulo `m`.
+ *      The return type is the largest between `x`'s and `m`'s type.
+ *
+ * All values have to be unsigned types, otherwise it will raise a compilation error.
```

Comment:
Replace with:

The function is only defined for unsigned types and statically rejects all others.